tools/CleanOutput.py

```diff
@@ -0,0 +1,77 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import os
+import re
+import logging
+
+
+def clean_output(input_file, output_dir):
+    """
+    Deletes all output files generated by a run with the `input_file` in
```

Comment:
I would prefer this to say "Deletes output files" or "Deletes specified output files", because it's not true that the script will generically delete all output.